app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
```

Comment:
Don't hesitate to use longer and more descriptive variable names, they are ususlly much more readable, e.g. `visitors_without_masks_count`